source/new-ia/by-role/developer/rest-services/start-a-process.rst

```diff
@@ -0,0 +1,77 @@
+:is-up-to-date: True
+:last-updated: 4.0.3
+
+:nosearch:
+
+=========================================
+Start an Activiti Process via Rest Script
+=========================================
+
+This cook book is designed to provide the most straight forward example of a CrafterCMS REST service being used to start an Activiti Process.
```

Comment:
I'm not sure this should be in our docs. It's not complete, we're not sure it still works and doesn't fit in the new IA at the moment. As a blog or plugin, this works, as docs, not sure.